### PR TITLE
Amina/84098 demo compare accounts table for CR

### PR DESCRIPTION
**Pull request by amina-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by thisyahlen-deriv — APPROVED** [review]

File: packages/cfd/src/Constants/cfd_compare_account_content.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    { label: localize('Add'), action: 'financial_bvi' },
+    { label: localize('Add'), action: 'financial_vanuatu' },
+    { label: localize('Add'), action: 'financial_labuan' },
+    { label: localize('Add'), action: 'derivx' },
+];
+export const eu__real_footer_button: TCompareAccountFooterButtonData[] = [
+    { label: localize('Add'), action: 'financial_maltainvest' },
+];
+
+export const preappstore_demo_cr_content: TCompareAccountContentProps[] = [
```
Comment (mahdiyeh-fs) [inline packages/cfd/src/Constants/cfd_compare_account_content.ts:173]: to be consistent with other variables I suggest to change it to
```suggestion
export const preappstore_cr_demo_content: TCompareAccountContentProps[] = [
```

File: packages/cfd/src/Constants/cfd_compare_account_content.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+export const cr_real_footer_buttons: TCompareAccountFooterButtonData[] = [
+    { label: localize('Add'), action: 'synthetic_svg' },
+    { label: localize('Add'), action: 'synthetic_bvi' },
+    { label: localize('Add'), action: 'financial_svg' },
+    { label: localize('Add'), action: 'financial_bvi' },
+    { label: localize('Add'), action: 'financial_vanuatu' },
+    { label: localize('Add'), action: 'financial_labuan' },
+    { label: localize('Add'), action: 'derivx' },
+];
+export const eu__real_footer_button: TCompareAccountFooterButtonData[] = [
```
Comment (mahdiyeh-fs) [inline packages/cfd/src/Constants/cfd_compare_account_content.ts:169]:
```suggestion
export const eu_real_footer_button: TCompareAccountFooterButtonData[] = [
```

File: packages/cfd/src/Constants/cfd_compare_account_content.ts [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                    localize('Stock indices'),
+                    localize('Commodities'),
+                    localize('Cryptocurrencies'),
+                ],
+            },
+        },
+    },
+];
+
+export const preappstore_demo_cr_footer_buttons: TCompareAccountFooterButtonData[] = [
```
Comment (mahdiyeh-fs) [inline packages/cfd/src/Constants/cfd_compare_account_content.ts:222]:
```suggestion
export const preappstore_c_demor_footer_buttons: TCompareAccountFooterButtonData[] = [
```

File: packages/cfd/src/Containers/mt5-compare-table-content.tsx [hunk 4]
```diff
@@ -285,6 +74,7 @@ const DMT5CompareModalContent = ({
             ? 4
             : available_accounts_keys.filter(key => key.startsWith('financial')).length || 1;
 
+    const is_preappstore_demo_cr_account = is_demo_tab && !is_eu && should_show_derivx;
```
Comment (mahdiyeh-fs) [inline packages/cfd/src/Containers/mt5-compare-table-content.tsx:77]:
```suggestion
    const is_preappstore_cr_demo_account = is_demo_tab && !is_eu && should_show_derivx;
```

**Review by mahdiyeh-fs — APPROVED** [review]

**Review by jim-deriv — APPROVED** [review]

**Comment by amina-deriv** [comment]
closing this PR since the changes are merged in https://github.com/matin-deriv/deriv-app/pull/72

**Pull request closed without merging**